src/coreclr/jit/lclvars.cpp

```diff
@@ -3402,7 +3402,7 @@ class LclVarDsc_BlendedCode_Less
             weight2 += 2 * BB_UNITY_WEIGHT;
         }
 
-        if (weight1 != weight2)
+        if (fabs(weight1 - weight2) > 0.01)
```

Comment:
This is the similar logic to in `fgProfileWeightsEqual` -- should we call that here so we have just one threshold for nearly equal weights?